backend/test/features/activity-delete.feature

```diff
@@ -5,51 +5,42 @@ Feature: Delete an object
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug          |
-      | bernd-das-brot|
-    And I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+      | spongebob     |
+    And we have the following post in our database:
     """
     {
       "@context": "https://www.w3.org/ns/activitystreams",
-      "id": "https://aronda.org/users/bernd-das-brot/status/lka7dfzkjn2398hsfd",
+      "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234s",
       "type": "Create",
-      "actor": "https://aronda.org/users/bernd-das-brot",
+      "actor": "http://localhost:4123/users/spongebob",
       "object": {
-          "id": "https://aronda.org/users/bernd-das-brot/status/kljsdfg9843jknsdf234",
-          "type": "Article",
-          "published": "2019-02-07T19:37:55.002Z",
-          "attributedTo": "https://aronda.org/users/bernd-das-brot",
-          "content": "Hi Max, how are you?",
-          "to": "https://www.w3.org/ns/activitystreams#Public"
+        "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234",
+        "type": "Article",
+        "published": "2019-02-07T19:37:55.002Z",
+        "attributedTo": "https://localhost:4123/api/users/spongebob",
+        "content": "Test content for deletion!",
+        "to": "https://www.w3.org/ns/activitystreams#Public"
       }
     }
     """
 
   Scenario: Deleting a post (Article Object)
-    When I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+    When I send a POST request with the following activity to "/inbox":
     """
     {
       "@context": "https://www.w3.org/ns/activitystreams",
-      "id": "http://localhost:4123/activitypub/users/karl-heinz/status/a4DJ2afdg323v32641vna42lkj685kasd2",
+      "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234s",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-06T23:09:16Z" title="Tuesday, May 7th 2019, 1:09:16 am +02:00">May 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -5,51 +5,42 @@ Feature: Delete an object
     Given our own server runs at "http://localhost:4123"
     And we have the following users in our database:
       | Slug          |
-      | bernd-das-brot|
-    And I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+      | spongebob     |
+    And I send a POST request with the following activity to "/inbox":
     """
     {
       "@context": "https://www.w3.org/ns/activitystreams",
-      "id": "https://aronda.org/users/bernd-das-brot/status/lka7dfzkjn2398hsfd",
+      "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234s",
       "type": "Create",
-      "actor": "https://aronda.org/users/bernd-das-brot",
+      "actor": "http://localhost:4123/users/spongebob",
       "object": {
-          "id": "https://aronda.org/users/bernd-das-brot/status/kljsdfg9843jknsdf234",
-          "type": "Article",
-          "published": "2019-02-07T19:37:55.002Z",
-          "attributedTo": "https://aronda.org/users/bernd-das-brot",
-          "content": "Hi Max, how are you?",
-          "to": "https://www.w3.org/ns/activitystreams#Public"
+        "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234",
+        "type": "Article",
+        "published": "2019-02-07T19:37:55.002Z",
+        "attributedTo": "https://localhost:4123/api/users/spongebob",
+        "content": "Test content for deletion!",
+        "to": "https://www.w3.org/ns/activitystreams#Public"
       }
     }
     """
 
   Scenario: Deleting a post (Article Object)
-    When I send a POST request with the following activity to "/activitypub/users/bernd-das-brot/inbox":
+    When I send a POST request with the following activity to "/inbox":
     """
     {
       "@context": "https://www.w3.org/ns/activitystreams",
-      "id": "http://localhost:4123/activitypub/users/karl-heinz/status/a4DJ2afdg323v32641vna42lkj685kasd2",
+      "id": "https://localhost:4123/api/users/spongebob/status/kljsddgnffgdgsx9bv8adg43jfdgkgnsdf234s",
```


Why `https` here? @Mastercuber :thinking:

Reply:
> [<img alt="Mastercuber" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mastercuber) **Authored by [Mastercuber](https://github.com/Mastercuber)**
_<time datetime="2019-05-10T20:47:56Z" title="Friday, May 10th 2019, 10:47:56 pm +02:00">May 10, 2019</time>_

---


I used https where I can. For some tests the service sends a request where the actual path, instead of the frontend prefix, is used. Also the response of the server is created with the `/api` prefix and the https protocol. To use the frontend port in the tests would need changes just for testing.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-13T10:42:10Z" title="Monday, May 13th 2019, 12:42:10 pm +02:00">May 13, 2019</time>_

---


Can you use `http` every time? We can assume that all our local web servers run locally, no `https` required. For testing, this is :ok: